acl_loader/main.py

```diff
@@ -518,9 +595,20 @@ def full_update(self):
         """
         for key in self.rules_db_info.keys():
             if self.current_table is None or self.current_table == key[0]:
-               self.configdb.mod_entry(self.ACL_RULE, key, None)
+                self.configdb.mod_entry(self.ACL_RULE, key, None)
+                """
+                Program for per front asic namespace also if present
+                """
```

Comment:
The `#` style should be used for comments except for docstrings.

Reply:
@daall Done. Will be using this guide-line from next time.